include/NAS2D/Mixer/MixerNull.h

```diff
@@ -13,7 +13,6 @@ namespace NAS2D
 		MixerNull(MixerNull&& other) = default;
 		MixerNull& operator=(const MixerNull& other) = default;
 		MixerNull& operator=(MixerNull&& other) = default;
-		virtual ~MixerNull() = default;

```

Comment:
This is inconsistent with other changes where `= default` was *added*. Did you mean to mark it override or leave it as-is and just got caught up in the sweeping changes, instead?

Reply:
That was intentional. It was done as it's own commit because it was different from the rest.

Since `MixerNull` is a derived class, there is no reason to write an explicit destructor if it's not adding anything to it. There is already a `default` `virtual` destructor from the base class. Better to just omit it entirely.

Reply:
If you only remove the destructor you are violating rule [CS.21: If you define or delete any default operation, define or delete them all](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-default-operation-define-or-delete-them-all) by leaving the rest.

See also: [Special Member Functions](https://en.wikipedia.org/wiki/Special_member_functions)

Since this class is designed to do nothing, recommend Rule of Zero.

Reply:
I agree. They should all be removed.